src/github.com/travis-ci/worker/processor_pool.go

```diff
@@ -46,6 +47,9 @@ func NewProcessorPool(hostname string, ctx gocontext.Context, hardTimeout time.D
 	}
 }
 
+// Each loops through all the processors in the pool and calls the given
+// function for each of them, passing in the index and the processor. The order
+// of the processors is the same for the same set of processors.
 func (p *ProcessorPool) Each(f func(int, *Processor)) {
```

Comment:
`Map` to me is for a function that converts each element in a collection and returns a new collection, so I think `Each` is better here. I was a little confused about the similar function in `ProviderConfig` being named `Map` when it didn't return anything.